doc/bucket-class-crd.md

```diff
@@ -1,73 +1,75 @@
+
+## Replication Policy
+A namespace bucket class defines a policy for namespace buckets.
+A bucket class will define a replication policy when an admin would like to replicate objects 
 within noobaa bucket to another noobaa bucket (source/destination buckets can be regular/namespace buckets).
 
 Replication policy will require the content of a JSON file which defines array of rules.
   - Each rule is an object contains rule_id, destination bucket and optional filter object that contains prefix field.
   - When a filter with prefix is defined - only objects keys that match prefix, will be replicated.
 
-Constraints:
-A backing-store name may appear in more than one bucket-class but may not appear more than once in a single bucket-class.
-The operator cli currently only supports a single tier placement-policy for a bucket-class.
-Upon creating regular buckets, the user will first need to create a placement-bucketclass which contains placemant policy.
-Upon creating namespace buckets, the user will first need to create a namespace-bucketclass which contains namespace policy.
-A namespace bucket class of type cache must contain both Placement-policy or Namespace-policy.
-A namespace bucket class of type single/multi must contain Namespace-policy.
-YAML must be used to create a bucket-class with a placement-policy that has multiple tiers.
-Placement-policy is case sensitive and should be of value (Mirror|Spread).
-Namespace-policy is case sensitive.
+## Constraints:
+- A backing store name may appear in more than one bucket class but may not appear more than once in a single bucket class.
+- The operator CLI currently only supports a single tier placement-policy for a bucket class.
+- Upon creating regular buckets, the user will first need to create a placement bucketclass which contains a placemant policy.
+- Upon creating namespace buckets, the user will first need to create a namespace bucketclass which contains namespace policy.
+- A namespace bucket class of type cache must contain both a placement and a namespace policy.
+- A namespace bucket class of type single/multi must contain Namespace-policy.
+- YAML must be used to create a bucket class with a placement-policy that has multiple tiers.
+- Placement policy is case sensitive and should be of value `Mirror` or `Spread` when more than one backingstore is provided.
+- Namespace policy is case sensitive.
 
-For more information on using bucket-classes from S3 see [S3 Account](s3-account.md).
-
-# Definitions
-
-- CRD: [noobaa.io_bucketclasses_crd.yaml](../deploy/crds/noobaa.io_bucketclasses_crd.yaml)
-- CR: [noobaa.io_v1alpha1_bucketclass_cr.yaml](../deploy/crds/noobaa.io_v1alpha1_bucketclass_cr.yaml)
+For more information on using bucket classes from S3 see [S3 Account](s3-account.md).
```

Comment:
I think you can remove this note - we cannot reach the file...
![Screenshot 2023-04-02 at 10 26 06](https://user-images.githubusercontent.com/57721533/229338841-0cae8a7e-749d-4ed7-a38a-a56364883d2d.png)

I also found that it was something that was planned to add:
https://github.com/noobaa/noobaa-operator/blob/c9254463b427fd3c4cdf2d0be26ab58559ad372c/doc/noobaa-account-crd.md?plain=1#L13